src/main/java/com/google/sps/servlets/GenerateServlet.java

```diff
@@ -108,6 +126,24 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     return filteredDestinations;
   }
 
+  /* Returns the filteredDestinations Set with the correct number of Destinations. */
```

Comment:
Could you also mention here that it chooses a random subset of destinations to meet that filtering requirement?